torch/_inductor/constant_folding.py

```diff
@@ -195,3 +203,60 @@ def constant_fold(gm, constraint_fn: Optional[Callable[[torch.fx.Node], bool]] =
     gm.graph.eliminate_dead_code()
     gm.graph.lint()
     gm.recompile()
+
+
+@torch.utils._python_dispatch._disable_current_modes()
+def constant_graph_tag(gm: torch.fx.GraphModule):
+    cf = ConstantFolder(gm, skip_constructors=True)
+    cf.run()
+
+    for node in gm.graph.nodes:
+        if (
+            node.op == "get_attr"
+            or node in cf.node_replacements
+            or node in cf.replaced_uses
+        ):
+            node.tag = CONST_MODULE_TAG
+        else:
+            node.tag = MODULE_TAG
+
```

Comment:
we typically put important metadata of nodes in `node.meta`. or we could maintain dictionaries ourself.